ISMIR2021.tex

```diff
@@ -151,12 +155,86 @@ \section{Introduction}
+    \item To make providing data as easy as possible for the other chairs, we attempted to support simplified data import formats, instead of the more comprehensive, original Midspace formats.
+    \item Once the conference start was very close, we switched over to manual data entry.
+    \item Because the registration system (which was not chosen by the tech chairs) did not offer a suitable API, authors and attendees had to be imported into Midspace manually by running custom-coded scripts.
+    \item The gathe.town map was created in a long process together with a graphic designer from korea that we payed for via Fiverr. The designer created the concept and designed the space with respect to our requirements (exptected number of attendees, etc.). He also helped us to fill in the sponsor area in gather town which resulted in many additional changes due to sponsor re-requests.
+    \item Events on \emph{Gater.town} were also manually entered via the map-making tool. This is a tedious process and can not be automated or synced to the conference schedule.
+    \item Zoom events where manually scheduled and we had to use multiple admin accounts to manage the events. Note that any changes to the schedule were not synced to the Zoom events and thus always leads to conflicts.
+\end{itemize}
+
+\subsection{Running the Conference}
+
+\begin{itemize}
+    \item A \href{https://docs.google.com/spreadsheets/d/1ENeS3DyxSxWQGRSsiMsHuddwmdlg6r_4tH6VTcvLgMg/edit?usp=sharing}{storyboard} was created for the entire conference with the complete details of each session - start and end time (timezone appropriate), session chairs, panelists (if any), platform used, direct link to the session (if any). Further, each session was assigned one tech-chair and a tech-volunteer that are required to be present at the session.
+    \item The main task of the tech-volunteers was to help session chairs and authors to seamlessly navigate the conference platforms. The tech-chairs were only a stand-by option for worst case scenarios---when admin access was required to make changes to the platform or when the tech-volunteer couldn't make it to the session.
+    \item As tech-volunteers we recruited 3-4 students (see~\ref{sssec:vol}) across each of the three timezones, that roughly correspond to Asia, Europe and US timezones. We had 12 volunteers in total.
+    \item A couple of weeks before the conference, all tech-volunteers were given a walkthrough of the conference platforms and a live demo of how to manage the platforms.
+    \item During the conference, the respective tech-chairs would remind the responsible tech-volunteers of the sessions a day earlier. 
+\end{itemize}
+
+\subsection{Lesson's Learned}
+
+\begin{itemize}
+    \item Negotiation of contracts, especially in the US, seem to have been way underestimated by us. Start that early, if things go bogus, include the board, they are more than willing to help out.
```

Comment:
maybe rephrase. The board was willing to help, but they certainly would have preferred to not be involved.